assets/css/events.css

```diff
@@ -90,23 +15,100 @@
     transition: background-color 0.5s, box-shadow 0.5s;
 }
 
-.event:hover {
-    box-shadow: 0px 3px 1px -2px rgba(0, 0, 0, 0.2), 
-    0px 2px 2px 0px rgba(0, 0, 0, 0.14), 
-    0px 1px 5px 0px rgba(0, 0, 0, 0.12);
-}
-
-.event:hover .event-details {
-    background-color: rgb(248, 248, 248);    
-}
-
-.event-figure {
-    height: 150px;
-}
-
-.event-img {
-    width: 100%;
-    height: 100%;
-    object-fit: cover;
-    position: center;
-}
+    .event:hover {
+        box-shadow: 0px 3px 1px -2px rgba(0, 0, 0, 0.2), 
+        0px 2px 2px 0px rgba(0, 0, 0, 0.14), 
+        0px 1px 5px 0px rgba(0, 0, 0, 0.12);
+    }
+
+    .event:hover .event-details {
+        background-color: rgb(248, 248, 248);    
+    }
+
+    .event-box {
+        display: block;
+        color: inherit;
+        width: 100%;
+        height: 100%;
+    }
+
+        .event-figure {
```

Comment:
This isn't a child of `.event-box` so it shouldn't be indented.

Reply:
Yes, but HTML-wise .event-figure is a child of .event-box